lib/workers/global/index.ts

```diff
@@ -141,6 +142,7 @@ export async function start(): Promise<number> {
       }
       const repoConfig = await getRepositoryConfig(config, repository);
       if (repoConfig.hostRules) {
```

Comment:
@rarkins, I've looked at this piece of code multiple times, and was left confused every time. Doesn't this mean that if there are no hostRules, it will lend them from whatever repo last did declare them? Is that a preferable behavior?

Reply:
True, we should probably clear them and then reapply. But I think it's impossible to have zero hostRules anyway, because we always add one for the platform